docs/sources/troubleshooting/support-bundles/index.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Learn how to send a support bundle to Grafana Labs support for troubleshooting
+keywords:
+  - grafana
+  - troubleshooting
+  - support
+  - bundles
+title: Send a support bundle to Grafana Labs support
+menutitle: Send a support bundle to support
+weight: 200
+---
+
+# Send a support bundle to Grafana Labs support
+
+When you encounter problems with your Grafana instance, you can send us a support bundle that contains information about your Grafana instance, including:
+
+- Grafana version
+- Installed plugins
+- Grafana configuration
+- Deployed database information and migrations
+
+## Available support bundle components
+
+- **Usage statistics**: Usage statistic for the Grafana instance
+- **User information**: A list of users of the Grafana instance
+- **Database and Migration information**: Database information and migration log
+- **Plugin information**: Plugin information for the Grafana instance
+- **Basic information**: Basic information about the Grafana instance (version, memory usage, and so on)
+- **Settings**: Settings for the Grafana instance
+- **SAML**: Healthcheck connection and metadata for SAML (only displayed if SAML is enabled)
+
+## Steps
+
+To generate a support bundle and send the support bundle to Grafana Labs via a support ticket:
+
+1. Click the Help icon.
+
+1. Click **Support Bundles**.
+
+   ![Support bundle panel](/static/img/docs/troubleshooting/support-bundle.png)
+
+1. Click **New Support Bundle**.
+
+1. Select the components that you want to include in the support bundle.
+
+1. Click **Create**.
+
+1. After the support bundle is ready, click **Download**.
+
+   Grafana downloads the support bundle to an archive (tar.gz) file.
+
+1. Attach the archive (tar.gz) file to a support ticket that you send to Grafana Labs Technical Support.
```

Comment:
@Jguer , what is the email address they should send the bundle to?

Reply:
Normally not an email but a support ticket (this line was copied from the Dashboard support section)

Reply:
As long as users know how to do this, I'm fine with it. The dashboard instructions are also not clear on this point. I had asked about this and didn't hear a response, so we went with what we had.